src/client/Client.ts

```diff
@@ -11,6 +11,7 @@ import path from 'path';
 import fs from 'fs';
 import cron from 'node-cron';
 import child_process from 'child_process';
+import * as event from '../../assets/event.json';
```

Comment:
Los json pueden importarse sin el `*`
```suggestion
import event from '../../assets/event.json';
```